app/main.py

```diff
@@ -1,2 +1,83 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, *args) -> None:
+        self.nodelist = []
+        self.capacity = 8
+        self.load_factor = 2 / 3
+        self.hash_table = [[i] for i in range(self.capacity)]
+        self.reinit(args)
+
+    def reinit(self, args: list) -> None:
+        while len(args) > self.capacity * self.load_factor:
+            self.capacity *= 2
+        self.hash_table = [[i] for i in range(self.capacity)]
+        for key_value in args:
+            key = key_value[0]
+            value = key_value[1]
+            current_node = hash(key) % self.capacity
+            for i in range(self.capacity):
+                correction = 0
+                if current_node + i >= self.capacity:
+                    correction = self.capacity - 1
+                hash_node = self.hash_table[current_node + i - correction]
+                if len(hash_node) == 1:
+                    hash_node.append(
+                        [hash(key), key, value]
+                    )
+                    break
```

Comment:
Use while loop because you have hash of current_node
Dict works with hash not as loop